ansible/roles/tomcat/tasks/install.yml

```diff
@@ -38,14 +38,14 @@
 # We don't use the xml module to avoid a dependency on lxml
 - name: set tomcat_version
   set_fact:
-    tomcat_version: "{{ tomcat_metadata.content | regex_findall('<version>(8.5.*)</version>', '\\1') | last }}"
```

Comment:
We're actually still recommending Tomcat 8.5 (see https://docs.killbill.io/latest/getting_started.html#_tomcat_installation_and_configuration). I suggest staying with this version for now.

Were you able to successfully test it on 10.x?